pkgs/tools/admin/exoscale-cli/default.nix

```diff
@@ -2,17 +2,17 @@
 
 buildGo114Package rec {
   pname = "exoscale-cli";
-  version = "1.12.0";
+  version = "1.14.0";
 
   src = fetchFromGitHub {
     owner  = "exoscale";
     repo   = "cli";
     rev    = "v${version}";
-    sha256 = "04ym7mfv565icj3lmd2nrvq9asawwmmzg09681pj9py61ws56bxr";
+    sha256 = "10sx542m6r6yq99nqn69bjfc6rv4nr9swi74rfccxb7dbifdrqpp";
   };
 
   goPackagePath = "github.com/exoscale/cli";
-  goDeps = ./deps.nix;
```

Comment:
How come this isn't needed anymore?

Reply:
I mean, is it a refactoring or is it necessary to make the new version build?

Reply:
If it's a refactoring, you could put these lines in a separate commit such that the first commit only consists of bumping the version and the hash.

Reply:
The upstream repo ships with all dependencies in `vendor/` which should have been used in the first nix package already, really.

Reply:
@dramaturg Ok, makes sense, thanks! So it's a refactoring. I'd recommend putting it in a separate commit.

Reply:
I refactored the changes into two separate commits.